bcipy/helpers/tests/resources/mock_session/parameters.json

```diff
@@ -645,24 +645,18 @@
     "type": "int"
   },
   "lang_model_type": {
-    "value": "UNIFORM",
+    "value": "MIXTURE",

```

Comment:
It might be a better default to use UNIFORM? I could be swayed either way. Whatever you choose, update the helpTip with the correct `Default:`

Reply:
I feel the default should be our best LM, which at this point is the mixture. I've updated the helpTip defaults accordingly